code/_onclick/hud/rendering/plane_master.dm

```diff
@@ -31,16 +31,18 @@
 		relay_render_to_plane(mymob, render_relay_plane)
 
 ///Things rendered on "openspace"; holes in multi-z
-/atom/movable/screen/plane_master/openspace_backdrop
-	name = "open space backdrop plane master"
+/atom/movable/screen/plane_master/openspace
+	name = "open space plane master"
 	plane = OPENSPACE_BACKDROP_PLANE
 	appearance_flags = PLANE_MASTER
 	blend_mode = BLEND_MULTIPLY
 	alpha = 255
-/atom/movable/screen/plane_master/openspace
-	name = "open space plane master"
-	plane = OPENSPACE_PLANE
-	appearance_flags = PLANE_MASTER
+
+/atom/movable/screen/plane_master/openspace/backdrop(mob/mymob)
+	. = ..()
```

Comment:
what are these changes for

Reply:
oops leftover change from downstream reverting that